Std/Data/List/Basic.lean

```diff
@@ -1125,7 +1125,7 @@ instance nodupDecidable [DecidableEq α] : ∀ l : List α, Decidable (Nodup l)
 /-- `eraseDup l` removes duplicates from `l` (taking only the first occurrence).
   Defined as `pwFilter (≠)`.
 
-    eraseDup [1, 0, 2, 2, 1] = [0, 2, 1] -/
+    `eraseDup [1, 0, 2, 2, 1] = [0, 2, 1]` -/
```

Comment:
this is supposed to be an indented code block, it should not have backticks around it (or it should use triple backtick before and after)

Reply:
Ok in the docs it appears to have the wrong font for me (and no links / coloured background). Is this a docgen bug or are you just saying i should triple backtick it?

Reply:
I think this doc was copied from lean 3, it is possible docgen4 doesn't handle indent code blocks. Assuming adding more indentation doesn't fix it, you should just triple backtick it (but don't put the `-/` on the same line as the end backticks, this triggers a comment bug in vscode).

Reply:
just took a look, it appears the issue is the indentation of the `Defined as` line, that should be flush left or else lean will pull 2 spaces off the whole doc comment and break the indent code block

Reply:
the backticks are still there